src/test/java/educationalproject/programmingstuff/tests_integrations/UserServiceImplTest.java

```diff
@@ -0,0 +1,56 @@
+package educationalproject.programmingstuff.tests_integrations;
+
+import educationalproject.programmingstuff.servicies.UserService;
+import educationalproject.programmingstuff.servicies.dto.UserCreateRequestDto;
+import educationalproject.programmingstuff.servicies.dto.UserResponseDto;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@SpringBootTest
+@Transactional  // To reviewer: check this out. I've google this by myself. Test won't rise up without this.
+                // Now explanation needed. I didn't google explanation yet :)
```

Comment:
Maybe that's because you forgot @AutoConfigureMockMvc?
Transactional is not for tests, we should examine it later:)

Reply:
Ah, stop, you tried to write a unit test... We haven't learn it yet, that's why you did it wrong:)